Drivers/Inc/Minions.h

```diff
@@ -91,7 +126,6 @@ void Toggle_Set(light_t light, State state);
 */
 State Toggle_Read(light_t light);
```

Comment:
Make this Lights_Toggle_Read along with:
Toggle_Set -> Lights_Toggle_Set,
Toggle_Bitmap_Read -> Lights_Toggle_Bitmap_Read,

Reply:
Done